packages/ETLoad.package/ETLSampleImageSegmentTest.class/instance/testFile..st

```diff
@@ -1,3 +1,9 @@
 testing
 testFile: aFile
-	Transcript show: aFile; cr.
\ No newline at end of file
+	| reference |
+	
+	reference := ETLProjectLoadHelper loadSampleImageSegmentReference: aFile.
+	
+	[ ETLSegmentLoader do: [ ETLProjectLoadHelper loadSampleImageSegment: aFile. ]
+			withLoadResultDo: [ :result | self assert: (result literalEqual: reference). ]. ] ensure: [
```

Comment:
What advantage does `literalEqual` have here? It's not quite clear, and should probably be explained in a short comment.